modules/cluster-logging-deploy-cli.adoc

```diff
@@ -12,10 +12,7 @@ You can use the {product-title} CLI to install the Elasticsearch and Cluster Log
 * Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
 requires its own storage volume.
 +
-Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and limits.
-The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the
-{product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
-memory setting though this is not recommended for production deployments.
+Elasticsearch is a memory-intensive application. By default, {product-title} installs three Elasticsearch node with memory requests and limits of 16G. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues related to Elasticsearch, you should add more Elasticsearch nodes rather than increasing the memory on exiting nodes.
```

Comment:
```suggestion
Elasticsearch is a memory-intensive application. By default, {product-title} installs three Elasticsearch nodes with memory requests and limits of 16 GB. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues that are related to Elasticsearch, add more Elasticsearch nodes to your cluster instead of increasing the memory on exiting nodes.
```